script.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
Is this file included by accident?

Reply:
As mentioned in the description ```Special info for reviewer:
I have temporarily added script.go file that generated logs with \. So to test this PR, run loki make devenv sources=loki and run go run script.go. You will then have access to filenameA="E:\Logs\try\test\test.log label that can be used for testing. For query field, you still need to escape \ by yourself, so the query will be {filenameA="E:\\Logs\\try\\test\\test.log}.

The script.go file will be removed after this PR is approved and before merging to master.```

Reply:
Yeah sorry, only saw it later in the PR description. Never mind.

Reply:
No worries! I just wanted to make testing of this PR easier for the one who will be testing it. But I'll definitely remove the file before merging. 🙂